Bottom: 2082cdb7ee0b5491341ea2a2934b717463f5fd5e
Top:    5503e0239fdd5d6fbf0b99198e749578bed1ce43
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-16 12:01:28 +0530

Refresh of sched-fair-tune-task-wake-up

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index a5cee6876263..f038f65ed006 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5888,11 +5888,11 @@ static int select_idle_cpu(struct task_struct *p, struct sched_domain *sd, int t
 #ifdef CONFIG_SCHED_SMT
 
 /* Define non-idle CPU/task as the one with the utilization >= 12.5% */
-#define is_zealous(util) ((util) > (1024 >> 3))
+#define is_zealous(util) ((util) > (100 >> 3))
 
 static inline bool is_background_task(struct task_struct *p)
 {
-	if (task_latency_lenient(p) && !is_zealous(task_util(p)))
+	if (task_latency_lenient(p) && (task_util(p) > (1024 >> 3)))
 		return true;
 
 	return false;
